src/app/components/pages/swagger-endpoint-page/swagger-endpoint-page.component.scss

```diff
@@ -53,9 +57,22 @@
     flex-basis: 50%;
 }
 
+.endpoint__example {
+
```

Comment:
is this future improvements or no longer needed?

Reply:
future improvements as there are several empty classes which I haven't styled to stay within deadlines. We can look at this as future broader review of front-end once we have more content generated in the app.